src/main/java/io/vertx/core/json/Json.java

```diff
@@ -125,6 +125,31 @@ public static String encodePrettily(Object obj) throws EncodeException {
     }
   }
 
+  /**
+   * Decode a given JSON string.
+   *
+   * @param str the JSON string.
+   *
+   * @return a {@link JsonArray}, {@link JsonObject}, {@link String}, ...etc if the content is an array, object, string, ...etc
```

Comment:
`a JSON element which can be {@link JsonArray}, ....`

Reply:
Done